nym-api/src/support/config/mod.rs

```diff
@@ -246,9 +245,7 @@ pub struct Base {
     #[zeroize(skip)]
     pub local_validator: Url,
 
-    // TODO dz Andrew added this here ?!
-    /// Socket address this api will use for binding its http API.
-    /// default: `0.0.0.0:8080`
+    /// Socket address Axum will use for binding its HTTP API.
     #[zeroize(skip)]
     #[serde(default = "default_http_socket_addr")]
     pub bind_address: SocketAddr,
```

Comment:
@jstuczyn  I couldn't find this config value being used anywhere, could you confirm it isn't?